### PR TITLE
feat(tse): Introduction of a Python Library to work with the Output of the TSE Simulations

### PR DESCRIPTION
**Pull request by schwepmo**
## Description

* With this PR we introduce a small python "library" in combination with a jupyter notebook, which provides basic functionality for reading and processing the data generated by TSE simulations

## Affected parts of the online documentation

* Readme was adjusted accordingly

## Definition of Done

**Prerequisites**

- [x] You have read CONTRIBUTING.md carefully.

**Required**

- [x] The title of this merge request follows the scheme `type(scope): description`  (in the style of [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary))
- [x] `origin/main` has been merged into your Fork.
- [x] Coding guidelines have been followed (see CONTRIBUTING.md).
- [x] All checks on GitHub pass.

### CODE REVIEW

File: evaluation/notebooks/usage_example.ipynb
Comment (kschrab) [inline evaluation/notebooks/usage_example.ipynb:1]: Would it be better to commit a clean notebook?

File: evaluation/requirements.txt
Comment (kschrab) [inline evaluation/requirements.txt:1]: Why is this file recognized as binary??

Comment (kschrab) [reply evaluation/requirements.txt:1]: Anyhow, the requirements.txt is really long. Is this a usual size? From looking into the code, you just use `matplotlib`, `numpy`, and `pandas`

File: evaluation/notebooks/usage_example.ipynb
Comment (schwepmo) [reply evaluation/notebooks/usage_example.ipynb:1]: This should allow to view the cell results from within github which I think is fine ([here](https://github.com/mosaic-addons/traffic-state-estimation/blob/tse-python-lib-and-general-fixes/evaluation/notebooks/usage_example.ipynb)), and doesn't require users to execute the notebook

File: evaluation/requirements.txt
Comment (schwepmo) [reply evaluation/requirements.txt:1]: > Why is this file recognized as binary??

Good question, I have no idea.



> Anyhow, the requirements.txt is really long. Is this a usual size? From looking into the code, you just use `matplotlib`, `numpy`, and `pandas`

And `jupyter` at least. I just generated using `python -m pip freeze -l > requirements.txt`. Nonetheless, I reduced the *requirements.txt* to the basics and tried re-configuring the venv and everything worked on my machine 🤷

File: evaluation/notebooks/usage_example.ipynb
Comment (kschrab) [reply evaluation/notebooks/usage_example.ipynb:1]: Ok, valid reason :)

**Pull request merged**